image_proc/CMakeLists.txt

```diff
@@ -60,6 +60,25 @@ ament_target_dependencies(debayer
 rclcpp_components_register_nodes(debayer "image_proc::DebayerNode")
 set(node_plugins "${node_plugins}image_proc::DebayerNode;$<TARGET_FILE:debayer>\n")
 
+add_library(resize SHARED
+  src/resize.cpp
+  )
+
+target_compile_definitions(resize
+  PRIVATE "COMPOSITION_BUILDING_DLL")
+
+ament_target_dependencies(resize
```

Comment:
I've pointed to this before, but please do 

`set(dependencies [list, of, dependencies])`

and then use `${dependencies}` as needed. I don't want several long lists of packages that can be easily forgotten to be updated